src/vivarium/framework/results/manager.py

```diff
@@ -69,6 +71,33 @@ def setup(self, builder: "Builder"):
 
         builder.value.register_value_modifier("metrics", self.get_results)
 
+    def on_post_setup(self, event: Event):
+        # update self._metrics to have all output keys
+        def create_measure_specific_keys(measure: str, stratifications: List[str]) -> None:
+            measure_str = f"MEASURE_{measure}_"
+            individual_stratification_strings = [
+                [
+                    f"{stratification.name.upper()}_{category}"
+                    for category in stratification.categories
+                ]
+                for stratification in sorted(
```

Comment:
Is there a reason this has to be sorted?

Reply:
The stratifications are sorted by the results context so this makes sure that the output keys in post setup and the output keys generated by gather_results match.